modules/virt-latency-checkup-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-running-cluster-checkups.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-latency-checkup-web-console_{context}"]
+= Running a latency checkup in the web console
+
+Run a latency checkup to verify network connectivity and measure the latency between two virtual machines attached to a secondary network interface.
+
+.Prerequisites
+
+* You must add a `NetworkAttachmentDefinition`` to the namespace.
```

Comment:
1. Redundant "\`" at the end of `NetworkAttachmentDefinition`.
2. NAD is not enough. The user must also create a bridge interface on the cluster nodes, otherwise the NAD won't connect the VM's secondary interface to any interface on the node, and the job will fail because the VMs won't start.
Please add a reference to https://76590--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-linux-bridge#virt-connecting-vm-to-linux-bridge (it is referenced in the `Additional Resources` section, but at this point of creating a NAD and initiating a checkup, the un-experienced user wouldn't know that this is what they are missing).

Reply:
Updated.